homeassistant/util/location.py

```diff
@@ -17,16 +17,35 @@
      'city', 'zip_code', 'time_zone', 'latitude', 'longitude',
      'use_fahrenheit'])
 
+DATA_SOURCE = ['https://freegeoip.io/json/', 'http://ip-api.com/json']
```

Comment:
@andruby thanks for providing the mirror. Our impact should be minimal, it is only used on initial set up to get some  defaults for the configuration